ophyd/device.py

```diff
@@ -719,9 +721,11 @@ class Device(BlueskyInterface, OphydObject, metaclass=ComponentMeta):
     _default_read_attrs = None
     # If `None`, defaults to `[]`
     _default_configuration_attrs = None
+    # If `None`, defaults to `{'fields': []}`
```

Comment:
I thought we agreed that the default behaviour would be `{}` in the case of no hints?